src/m365/aad/commands/m365group/m365group-remove.ts

```diff
@@ -69,29 +74,18 @@ class AadM365GroupRemoveCommand extends GraphCommand {
   }
 
   public async commandAction(logger: Logger, args: CommandArgs): Promise<void> {
-    const removeGroup = async (): Promise<void> => {
-      if (this.verbose) {
-        await logger.logToStderr(`Removing Microsoft 365 Group: ${args.options.id}...`);
-      }
+    if (this.verbose) {
+      await logger.logToStderr(`Removing Microsoft 365 Group: ${args.options.id}...`);
```

Comment:
We placed this comment inside the `removeGroup` function so that it's displayed only after the user confirmed that they want to delete the group. Let's keep it that way and move it back to the function.